src/constants.nim

```diff
@@ -70,4 +70,19 @@ proc hasLogLevelOption*(): bool =
 proc runtimeLogLevelSet*(): bool =
   return existsEnv(RUN_TIME_PREFIX & "_LOG_LEVEL") or hasLogLevelOption()
 
+proc runtimeLogLevel*(): string =
+  const
+    upperCaseLogLevelCliPrefix = "--LOG_LEVEL="
+    lowerCaseLogLevelCliPrefix = "--log-level="
+    runtimeLogLevelEnvVar = RUN_TIME_PREFIX & "_LOG_LEVEL"
+
+  for p in cliParams:
+    if p.startswith(lowerCaseLogLevelCliPrefix) or p.startsWith(upperCaseLogLevelCliPrefix):
+      return p[(lowerCaseLogLevelCliPrefix.len)..^1]
+
+  if existsEnv(runtimeLogLevelEnvVar):
+    return getEnv(runtimeLogLevelEnvVar)
+
+  return desktopConfig.logLevel
```

Comment:
I don't think it's needed.
`desktopConfig.logLevel` already returns the runtime value.